packages/gestalt/src/Masonry/__integration__/.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "globals": {
```

Comment:
@christianvuerings i think this can be avoided, its baked into eslint now, you just need to add `.integration.js` to the overrides for jest env

Reply:
Good catch - I'll do this in a follow up PR. I'd like to get this merged ASAP.

Reply:
@chrislloyd @christianvuerings check out #193 which updates based on the instructions [here](https://github.com/smooth-code/jest-puppeteer#configure-eslint)